src/main/java/com/linecorp/armeria/client/tracing/TracingRemoteInvoker.java

```diff
@@ -59,7 +60,8 @@ protected TracingRemoteInvoker(RemoteInvoker remoteInvoker, Brave brave) {
                                       Method method, Object[] args) throws Exception {
 
         // create new request adapter to catch generated spanId
-        final InternalClientRequestAdapter requestAdapter = new InternalClientRequestAdapter(method.getName());
+        final InternalClientRequestAdapter requestAdapter = new InternalClientRequestAdapter(
+                Endpoint.create(uri.toString(), 0, 0), method.getName());
```

Comment:
@synk It seems like `ClientRequestAdapter` now allows specifying the information about the endpoint. Because we do not know about the IP address and port number of the endpoint at this point, I just specified `0`. As you see, I specified the full URI of as a service name. Would this be OK?

By the way, I guess `Endpoint` API may change in the near future because it doesn't seem right not to support IPv6 addresses.